### DISCUSSION TITLE
Bug: Disable scan list while channel is busy

### DISCUSSION THREAD

**Discussion by oisact** [AubsUK/uv-k5-firmware-custom#8] [General]
I have several scanlists. One is of US NOAA radio. I enabled that scanlist and it stopped on an active channel. Then if I enable some other scanlist and disable that scanlist, the other scanlist will not become active until the current channel no longer has any activity.

It seems like disabling a scanlist should take affect even if it is on an active channel, leaving the active channel because its scanlist is no longer active.

**Comment by AubsUK** [comment]
Thanks for that, I think you're right.  What I'm understanding is if you are on an active channel and disable the ScanList the active channel is a member of, then it should continue scanning.  I'll look at that.

FYI - If you're on an active channel, you can press `*` and it'll temporary lock that channel out from being scanned until you power cycle the radio.

**Reply by oisact** [reply]
Yes that's correct. Probably not a big deal, as it mainly affects scan lists containing broadcast stations that never stop transmitting.

**Reply by AubsUK** [reply]
I agree, if you press the `up` or `down` key after disabling the ScanList, it should continue scanning on the next ScanList.

But I take the point if the button is pressed and the radio is receiving from a channel in the ScanList that was just disabled, it should continue scanning.

**Comment by AubsUK** [comment]
Added in [v00.05](https://github.com/AubsUK/uv-k5-firmware-custom/releases/tag/v00.05).
